### PR TITLE
Re-enabled popup controls and snippets.

### PR DESCRIPTION
**Pull request by azaslonov**
In this change we're gettin back the snippet for popup-based Menu and also adding the new "Popups" section to allow managing the popups.

### CODE REVIEW

**Comment by f-alizada** [comment]
The PR bumps the version of the paperbits/style. Which results the error of fetching fonts, should it be added somewhere in the documentation how to retrieve/use?

File: src/libraries/block-snippets.json [hunk 1]
```diff
@@ -1581,6 +1594,146 @@
...
+                                                "horizontal": "between"
+                                            }
+                                        }
+                                    },
+                                    "padding": {
+                                        "xs": {
+                                            "top": 10,
+                                            "bottom": 10,
+                                            "left": 10,
+                                            "right": 10
```
Comment (JMach1) [inline src/libraries/block-snippets.json:1700]: TBH I'm not familiar with this kind of UI definition, but could not be this simplify for something like: padding: { xs: 10 }?

Comment (azaslonov) [reply src/libraries/block-snippets.json:1700]: Yes, that would be simpler, but shortcuts not supported by the style compiler yet. Later on, we'll surely simplify it.

**Comment by azaslonov** [comment]
> The PR bumps the version of the paperbits/style. Which results the error of fetching fonts, should it be added somewhere in the documentation how to retrieve/use?

I have added a placeholder for Google Fonts API key and sent the steps to the content team to produce the documentation.

**Review by ygrik — APPROVED** [review]

**Pull request merged**